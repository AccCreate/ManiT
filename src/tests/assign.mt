a = 4;
b = "hi";
c = true;

print(a);
print(b);
print(c);
