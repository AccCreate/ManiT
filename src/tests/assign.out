4
hi
1
